src/web/lib/components/ThemeCustomBackgroundPicker/index.js

```diff
@@ -129,12 +141,7 @@ export class ThemeCustomBackgroundPicker extends React.Component {
                   <React.Fragment>
                     <Modal>
                       <ul className="errors">
-                        {errors.tooLarge && (
-                          <li>The image is too large. (1MB maximum size)</li>
-                        )}
-                        {errors.wrongType && (
-                          <li>The file is not an accepted image type.</li>
-                        )}
+                        {mapErrors(errors).map((error, i) => <li key={i}>{error}</li>)}
```

Comment:
Is `key={i}` doing anything useful? If not, let's remove it for simplicity.

Reply:
this is required by react. when mapping over items you need a key. do you prefer another name? or i can try using the error name..?

Reply:
I wasn't aware of the `key` requirement (https://reactjs.org/docs/lists-and-keys.html#keys), thanks for the explanation.